cluster/charts/rook-ceph/sample-cephcluster.yaml

```diff
@@ -0,0 +1,92 @@
+#
+# Sample CephCluster to customise and deploy after you have deployed the rook-ceph operator
```

Comment:
I'm against adding another example Ceph Cluster yaml. There are already ten or more examples in the docs and some in file form available.
It would be better to mention these points to look out for in the `NOTES.txt` and/or Rook Helm Chart Ceph Operator docs.

Reply:
The notes at the top are the main thing. That's what wasted hours for me and @MartinForReal. It is unexpected that an operator-pattern chart is no self-sufficient; in the sense that once deployed this chart can't process CephCluster resources without extra RBAC manifests.

I did see example RBAC on the website, but disregarded it as for people deploying from manifests and *not* using the operator chart. I want to make sure other don't have the same new-user experience.

Reply:
Oh, and then because there are some many different examples of the RBAC on the website. I picked up the wrong one initial, and it didn't work with the chart version. That's why I wanted to include a version-matched example in the chart.